src/components/transport_manager/include/transport_manager/android_ipc/android_transport_adapter.h

```diff
@@ -33,26 +33,37 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_BLUETOOTHLE_BLUETOOTHLE_TRANSPORT_ADAPTER_H_
-#define SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_BLUETOOTHLE_BLUETOOTHLE_TRANSPORT_ADAPTER_H_
+#ifndef SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_ANDROID_IPC_ANDROID_TRANSPORT_ADAPTER_H_
+#define SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_ANDROID_IPC_ANDROID_TRANSPORT_ADAPTER_H_
 
 #include "transport_manager/transport_adapter/transport_adapter_impl.h"
 
 namespace transport_manager {
 namespace transport_adapter {
 
+ enum class AndroidTransportType {
+   BLE,
+   BT
```

Comment:
@AKalinich-Luxoft Thanks, maybe its make sense, but seems we already have word "transport" in enum name of these types - `AndroidTransportType`
so when using we type something like:
`AndroidTransportType::BLE`    
On my opinion in case `AndroidTransportType::TRANSPORT_BLE` we have duplication of  "transport"

Reply:
Yes, let's keep it as is then